src/acl/Acl.h

```diff
@@ -109,14 +111,14 @@ typedef enum {
 } aclMatchCode;
 
 /// \ingroup ACLAPI
-/// ACL check answer; TODO: Rename to Acl::Answer
-class allow_t
+/// ACL check answer
+class Acl::Answer
```

Comment:
Squid code style is to use an explicit namespace wrapper around the entire code block, as you can see around the code at lines 24-35 above. This helps our automated code processing tools get namespace attributions correct.